spec/acceptance/nodesets/default.yml

```diff
@@ -1,10 +1,10 @@
 HOSTS:
-  centos-64-x64:
+  debian-73-x64:
```

Comment:
I changed this because, as of right now, `ruby-hocon` does not support Ruby 1.8, but that's what's installed by default on cent 6.

I'm not sure if we want to support Ruby 1.8 or not. If so, some work will need to be done on `ruby-hocon` (I took a stab at this earlier, seemed like it would be kind of a nightmare). If not, we should remove nodesets for nodes that have Ruby 1.8 or below installed.